OpenRA.Game/Map/Map.cs

```diff
@@ -252,7 +254,7 @@ public class Map : IReadOnlyFileSystem
 
 		internal Translation Translation;
 
-		public static string ComputeUID(IReadOnlyPackage package)
```

Comment:
IMO it would be better if we kept `public static string ComputeUID(IReadOnlyPackage package)` which calls `ComputeUID(package, GetMapFormat(package))`. `GetMapFormat` can then be kept as a private method avoiding the need to leak internal implementation details to the outside world.

Reply:
sure